torch/_inductor/ir.py

```diff
@@ -4681,17 +4681,11 @@ def create(cls, x: "TensorBox", tag: str, ranks: List[int], group_size: int):
         )
 
     def codegen_collective(self, wrapper, output_name, input_names):
-        wrapper.writeline(
```

Comment:
Are these changes intentional?

Reply:
Yes, these were bugs in the previous implementation that were calling two allgathers for every allgather node :)